### ISSUE TITLE
Dgs-dxwindow doesn't close

### ISSUE THREAD

**Issue by ERAGON007** [thisdp/dgs#120]
Reproduce:
```
local window = dgsCreateWindow(.....)
dgsSetProperty(window, "dontClose", true)


addEventHandler("onDgsWindowClose", root,
function()
    local dontClose = dgsGetProperty(source, "dontClose")
    if (dontClose) then
        cancelEvent()
        dgsAlphaTo(source, 0.0, "Linear", 350)
        setTimer(function(elem)
            dgsSetVisible(elem, false)
            dgsSetAlpha(elem, 1.0)
        end, 750, 1, source)
    end
end
)
```

Try to close the window, it will close and the animation works.
But when you set window visible to true again and try to close it, it won't trigger and won't close

**Comment by xLive** [comment]
Yes, it was fixed in https://github.com/thisdp/dgs/commit/8496cf71258776940e3b88f73c2388c97a5af866 make sure you're using the latest version

**Comment by ERAGON007** [comment]
I have dgs 3.519 installed

**Comment by xLive** [comment]
> I have dgs 3.519 installed
It was fixed in [3.520](https://github.com/thisdp/dgs/releases/tag/3.520)

**Comment by ERAGON007** [comment]
My bad then, thanks!
